src/Toolchain/ToolchainProvider.ts

```diff
@@ -135,19 +135,19 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
   install() {
     const notifyInstalled = (toolchainEnv: ToolchainEnv, toolchain: Toolchain) => {
       const name = `${toolchain.info.name}-${toolchain.info.version ?.str()}`;
-      vscode.window.showInformationMessage(`Install ${name} successfully`);
+      vscode.window.showInformationMessage(`${name} installation was successful.`);
```

Comment:
How about this? To make sure that we are the subject.

```suggestion
      vscode.window.showInformationMessage(`Installed ${name} successfully.`);
```